src/Tools/Source/RunTests/Program.cs

```diff
@@ -405,6 +405,7 @@ private static void CheckTotalDumpFilesSize()
                 currentTotalSize += fileSizeInMegabytes;
                 if (currentTotalSize > MaxTotalDumpSizeInMegabytes)
                 {
+                    ConsoleUtil.WriteLine($"Deleting {dumpFile} because we have exceeded our total dump size of {MaxTotalDumpSizeInMegabytes} megabytes.");
```

Comment:
```suggestion
                    ConsoleUtil.WriteLine($"Deleting '{dumpFile}' because we have exceeded our total dump size of {MaxTotalDumpSizeInMegabytes} megabytes.");
```

Reply:
Good suggestion, will merge now and fix to get this in soon.